ui-v2/app/styles/base/animation/index.scss

```diff
@@ -0,0 +1,13 @@
+%blink-in-fade-out {
+  transition-property: opacity;
+  transition-duration: 0.1s;
+  transition-timing-function: linear;
+  opacity: 0;
+  pointer-events: none;
+}
+%blink-in-fade-out-active {
+  transition-duration: 0;
+  transition-duration: unset;
+  opacity: 1;
+  pointer-events: auto;
+}
```

Comment:
Starting to add handy animations to my base files. For reuse elsewhere in my codebase.